CMakeLists.txt

```
@@ -28,6 +28,7 @@ option(GVS_BUILD_TESTS "Build unit tests" OFF)
 option(GVS_USE_DEV_FLAGS "Compile with all the flags" OFF)
 option(GVS_UPDATES_DISCONNECTED "Update third party repos" OFF)
 option(GVS_FORMAT_ON_BUILD "Format the code when building" OFF)
+option(GVS_ENABLE_CUDA "Use CUDA if it's available" ON)
```

This should probably be `GVS_USE_CUDA` since that's what you use below?
```suggestion
option(GVS_USE_CUDA "Use CUDA if it's available" ON)
```

---

Omg! Fixed.